app/experimenter/nimbus-ui/src/lib/visualization/constants.ts

```diff
@@ -202,9 +202,10 @@ export const CONVERSION_METRIC_COLUMNS = [
   {
     name: "Conversions / Total Users",
     displayType: DISPLAY_TYPE.CONVERSION_COUNT,
+    branchComparison: BRANCH_COMPARISON.ABSOLUTE,
   },
   {
-    name: "Conversion Rate",
+    name: "Absolute Conversion Rate",
```

Comment:
Hmm, I added `branchComparison: BRANCH_COMPARISON.ABSOLUTE` since I changed `branchComparison` to be required for passing into `TableVisualizationRow` and I thought the [map at the top of that component](https://github.com/mozilla/experimenter/pull/6313/files#diff-59a511b5ef280cddb7a1aa227d06514c16304b43db64f9ca08df59eaaff702b8L31) referenced "absolute," but it looks like this is different than "primary metrics" so I'll look into this a bit further 👀

Reply:
OK so, circling back here. We're not actually using `TableMetricConversion` on the Results page right now, but this is what it looks like in Storybook on `main`:

<img width="1618" alt="image" src="https://user-images.githubusercontent.com/13018240/131582596-97b81f53-9f09-4c1d-869c-b23505d5b1ae.png">

If I set the `branchComparison` here to `uplift`, we get this:
<img width="1443" alt="image" src="https://user-images.githubusercontent.com/13018240/131582694-ddb699bd-acb5-4784-bdd8-1aa7180a79e1.png">

Then setting it to `absolute`, it's looking as it should:
<img width="1479" alt="image" src="https://user-images.githubusercontent.com/13018240/131582766-f79c15b6-c80e-4c0a-8cea-30c8927a68f0.png">

I'll leave the branch comparison name to `absolute` since it renders the table as it did prior, but, I wonder if in your comment "It's conversions / total users" you were referring to the first column and not the second. I kind of feel like it should have the word "Absolute" as I changed it [here](https://github.com/mozilla/experimenter/pull/6313#discussion_r698903599)? But I can easily change it too, if we prefer "Conversion Rate". 👍

Reply:
Marina gave the thumbs up on using the word "Absolute" here.